### PR TITLE
DBZ-3683 Capture changes made by connector user & document that SYS/SYSTEM changes are not captured

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-3683

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/oracle.adoc [hunk 1]
```diff
@@ -1079,6 +1079,13 @@ ALTER DATABASE ADD SUPPLEMENTAL LOG DATA;
 The {prodname} Oracle connector requires that users accounts be set up with specific permissions so that the connector can capture change events.
 The following briefly describes these user configurations using a multi-tenant database model.
 
+[WARNING]
+====
+This user account should be used solely by the {prodname} Oracle connector to connect to the Oracle database and capture changes.
+This account should not be used to create tables or modify data as any change performed by this account will not be captured by the connector.
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/oracle.adoc:1085]: Why is this the case actually?

Comment (Naros) [reply documentation/modules/ROOT/pages/connectors/oracle.adoc:1085]: This was how the contribution originally came from Navis.  

Just a guess, but I suspect they were relying on the connector user to write records to the `HistoryRecorder` interface which they used to store all read LogMiner events in a database table for debuggin purposes.  It was likely easier to make this distinction in the LogMiner query than to add table exclusions I suppose.  Keep in mind this was all before we retrofitted the LogMiner query to apply table/schema include/exclude lists.

We can certainly look at relaxing this, but I think we should at a minimum keep `SYS` and `SYSTEM`.  wdyt?

**Comment by Naros** [comment]
Moved to draft as we want to relax the restrictions on the connector user but keep the restrictions on `SYS` and `SYSTEM`.

**Comment by Naros** [comment]
@gunnarmorling I've relaxed the restriction on the connector user as requested and I've updated the documentation both with two different warnings in different sections as well as a "when things go wrong" direct link we can use or users may see when looking at the connector's ToC.

**Review by gunnarmorling — APPROVED** [review]
LGTM, merging. Thx!

**Pull request merged**